lib/web_console/request.rb

```diff
@@ -32,6 +32,49 @@ def acceptable_content_type?
       content_type.blank? || content_type.in?(acceptable_content_types)
     end
 
+    # Returns whether the request includes "application/vnd.web-console"
```

Comment:
I don't think we need those methods at all (see the comment below), however I do have one more problem with them: I think you made them public, only so you can test them. While testing is good, and we should do more of it here, I don't think easier testing a good enough reason to blow up an interface. In this case, you could have tested them through `acceptable_required_version?`.